main.cpp

```diff
@@ -0,0 +1,67 @@
+#include <stdio.h>
+#include <SDL2/SDL.h>
+#include <SDL2/SDL_timer.h>
+
+int main(int argc, char *argv[])
```

Comment:
Please stay with the same style.
```suggestion
int main(int argc, char *argv[]) {
```

Reply:
revised